style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: purple;
+}
```

Comment:
- [ ] Here you are using an element selector, but you are supposed to use a class selector. Please add a class in your heading and use that class to style your heading.

Reply:
Noted. A class name has now been used to style the heading.

Reply:
https://github.com/jmoseka/Hello-World/blob/47d1e57c2535ede4f1615d7e99b867f5c1c63643/style.css

Reply:
st [style.css #1](https://github.com/jmoseka/Hello-World/blob/47d1e57c2535ede4f1615d7e99b867f5c1c63643/style.css)

 [index.html #11](https://github.com/jmoseka/Hello-World/blob/47d1e57c2535ede4f1615d7e99b867f5c1c63643/index.html)

Reply:
[style.css #1](https://github.com/jmoseka/Hello-World/blob/47d1e57c2535ede4f1615d7e99b867f5c1c63643/style.css)

 [index.html #11](https://github.com/jmoseka/Hello-World/blob/47d1e57c2535ede4f1615d7e99b867f5c1c63643/index.html)